src/framework/application.js

```diff
@@ -1695,6 +1695,7 @@ Object.assign(Application.prototype, {
      * @param {number|null} [settings.render.skybox] - The asset ID of the cube map texture to be used as the scene's skybox. Defaults to null.
      * @param {number} settings.render.skyboxIntensity - Multiplier for skybox intensity.
      * @param {number} settings.render.skyboxMip - The mip level of the skybox to be displayed. Only valid for prefiltered cubemap skyboxes.
+     * @param {number[]} settings.render.skyboxRotation - Rotation of skybox.
```

Comment:
should be Quat instead of number[] ?

Reply:
this is correct - I had it as Quat before, but Will pointed out that render settings done as number arrays, and are objects initialized from arrays - look at the example of Color render settings.